### PR TITLE
Added changelog for files service 1.1.0

### PR DESCRIPTION
**Pull request by chad-pelly**

### CODE REVIEW

File: changelog-generator/content/changelogs/20230809_files_service-1-1-0.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+**⚒️ Improvements**
+* Error messages
+  * Improved the error message for creating a file without a request body
+
+
+**🐞 Bugs Fixed**
+
+* File Tags
```
Comment (AaronTrippaers) [inline changelog-generator/content/changelogs/20230809_files_service-1-1-0.md:19]: I think bug fixes don't have this title like line. Normally it is just a list without indentations?
Not sure if it matters a lot.
Screenshot 2023-08-09 at 10 00 46

Comment (AaronTrippaers) [reply changelog-generator/content/changelogs/20230809_files_service-1-1-0.md:19]: I also think the improvements can be just one line :

* Improved the error message for creating a file without a request body

The * Error messages is unnecessary

Comment (chad-pelly) [reply changelog-generator/content/changelogs/20230809_files_service-1-1-0.md:19]: I just copied the release notes and changed the format slightly, I'll out it to this format ^^

Comment (AaronTrippaers) [reply changelog-generator/content/changelogs/20230809_files_service-1-1-0.md]: Yeah, I should have noticed during the review of the release notes on confluence. 😬 Here it stuck out a bit more.

**Review by AaronTrippaers — APPROVED** [review]

**Pull request merged**